packages/block-editor/src/components/block-list/block-mobile-floating-toolbar.native.js

```diff
@@ -10,21 +10,23 @@ import styles from './block-mobile-floating-toolbar.scss';
 /**
  * WordPress dependencies
  */
+import { compose, withPreferredColorScheme } from '@wordpress/compose';
 import { createSlotFill } from '@wordpress/components';
 
 const { Fill, Slot } = createSlotFill( 'FloatingToolbar' );
 
-function FloatingToolbar( { children } ) {
+const FloatingToolbarFill = ( { children, getStylesFromColorScheme } ) => {
```

Comment:
Note that there is also a `usePreferredColorScheme` hook. I'm fine with either option, but the hook seemed like a better fit for a functional component, so just raising awareness of it 😁

Reply:
You're right that there is a hook called `usePreferredColorScheme` but I would need to duplicate logic from here since the hook provides only 'dark' or 'light' string.
https://github.com/WordPress/gutenberg/blob/master/packages/compose/src/higher-order/with-preferred-color-scheme/index.native.js#L9-L18